packages/libs/src/services/identity/IdentityService.ts

```diff
@@ -125,9 +125,11 @@ export class IdentityService {
   }
 
   async setAuthFn(obj: Data) {
+    const headers = await this._signData()
```

Comment:
this generates the header signatures using the web3Manager wallet, this is why we need the wallet obj here
https://github.com/AudiusProject/hedgehog/pull/58

Reply:
so here if we had wallet as an arg, we would do wallet.sign()?

Reply:
exactly, we could also set it as the hedgehog wallet for web3provider here too and then call this function afterward